### PR TITLE
Rename minionssh to sshminion

### PR DESCRIPTION
**Pull request by rjmateus**
Signed-off-by: Ricardo Mateus <rmateus@suse.com>

## What does this PR change?

Rename minionssh to sshminion. This is apply only to internal sumaform names to minimize impact on existing jenkins jobs specified in sumaform-test-runner project.

Any changes to machine names would have a huge impact on DNS names for those machines, making the risk to high when comparing to the benefits we would have.
 
This PR relates with the following PR: https://gitlab.suse.de/galaxy/sumaform-test-runner/merge_requests/202

Jenkins Build:
https://ci.suse.de/view/Manager/view/Manager-3.2/job/manager-3.2-cucumber-pipeline-NUE/678/

### CODE REVIEW

**Comment by srbarrios** [comment]
LGTM

File: modules/cucumber_testsuite/main.tf [hunk 1]
```diff
@@ -228,7 +228,7 @@ module "ctl" {
   minion_configuration    = module.min-sles12sp4.configuration
   centos_configuration    = contains(local.hosts, "min-centos7") ? module.min-centos7.configuration : { hostnames = null, ids = null }
   ubuntu_configuration    = contains(local.hosts, "min-ubuntu1804") ? module.min-ubuntu1804.configuration : { hostnames = null, ids = null }
-  minionssh_configuration = contains(local.hosts, "minssh-sles12sp4") ? module.minssh-sles12sp4.configuration : { hostnames = null, ids = null }
+  sshminion_configuration = contains(local.hosts, "minssh-sles12sp4") ? module.minssh-sles12sp4.configuration : { hostnames = null, ids = null }
```
Comment (Bischoff) [inline modules/cucumber_testsuite/main.tf]: OK, I am taking note you're not changing hosts names like "minssh-sles12sp4". That would have needed quite a lot of DHCP and DNS changes.

Comment (rjmateus) [reply modules/cucumber_testsuite/main.tf]: Correct Eric, I leaving that for the moment we change DNS and DHCP.
When we change hostnames, e syncronize in here also.

Comment (Bischoff) [reply modules/cucumber_testsuite/main.tf]: All fine Ricardo :smiley_cat:  .

**Pull request merged**